layouts/css/layout/_dark-theme.scss

```diff
@@ -146,4 +146,11 @@
       background-color: $dark-code-background;
     }
   }
+
+  .light-icon {
+    display: none;
```

Comment:
This and display: inline should be moved to [utils](https://github.com/nodejs/nodejs.org/blob/main/layouts/css/_utils.scss) like:

```css
.d-inline {
  display: inline !important;
}

.d-none {
  display: none !important;
}
```

That way you can reuse them across the codebase.